test/fixtures/output/network/private-link/attachment/connection/describe-aws-json.golden

```diff
@@ -2,7 +2,7 @@
   "id": "plattc-111111",
   "name": "aws-plattc",
   "cloud": "AWS",
-  "private_link_attachment_id": "platt-111111",
+  "private_link_attachment_id": "plattc-111111",
```

Comment:
why do we need this change? `private-link-attachment-id` should be of the form `platt-xxx` unless this is something different 🤔